packages/dashboard-backend/src/devworkspace-client/services/api/serverConfigApi.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2018-2021 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+
+import * as k8s from '@kubernetes/client-node';
+import { IServerConfigApi } from '../../types';
+import { createError } from '../helpers';
+import { api } from '@eclipse-che/common';
+
+const CUSTOM_RECOURSE_DEFINITIONS_API_ERROR_LABEL = 'CUSTOM_RESOURSE_DEFINITIONS_API_ERROR';
+
+const GROUP = 'org.eclipse.che';
+const VERSION = 'v1';
+const PLURAL = 'checlusters';
+
+const NAME = 'eclipse-che';
+const NAMESPACE = 'eclipse-che';
```

Comment:
The PR works great, but just wondering, would there be situations where the name and namespace are not `eclipse-che`?

Reply:
I have improved it.

Reply:
![Знімок екрана 2021-11-26_16_45_28](https://user-images.githubusercontent.com/6310786/143601014-627ce779-0b19-42ee-87e2-fa2700896788.png)

Reply:
It will require $CHE_NAMESPACE(kubernetes namespace where Che Cluster should be looked into) and $CHE_CRD_OBJECT_NAME(kubernetes CRD object name).

@dkwon17 WDYT?

Reply:
Looks good. On the Che operator side, I plan to set the environment variables like so: https://github.com/dkwon17/che-operator/commit/15afe05a8cb3ae6634722d14c18f2cafcc2fb09f#diff-f26d61ce8d3fc7ba6d48229eb77344a2be0d209ac7b4eeb9f850e4d05873adac